js/controller.js

```diff
@@ -1452,14 +1452,14 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     onChangeClosedCaptionLanguage: function(event, language) {
       var availableLanguages = this.state.closedCaptionOptions.availableLanguages;
-
       //validate language is available before update and save
-      if (language && availableLanguages && _.contains(availableLanguages.languages, language)) {
+        if (language && availableLanguages && (_.contains(availableLanguages.languages, language) || language === OO.CONSTANTS.CLOSED_CAPTIONS.NONE)) {
```

Comment:
We'll need to go through the html5-common update process to update OO.CONSTANTS.CLOSED_CAPTIONS with the NONE value.